api/project.py

```diff
@@ -7,6 +7,29 @@
 from controller.attribute import manager as attribute_manager
 from submodules.model import exceptions
 
+import os
+from controller.tokenization import tokenization_service
+from submodules.model import enums, events
+from submodules.s3.controller import bucket_exists, create_bucket
+from util import doc_ock, notification
+from controller.transfer.record_transfer_manager import import_records_and_rlas
+from controller.transfer.manager import check_and_add_running_id
+from controller.auth import manager as auth_manager
+from controller.project import manager as project_manager
+from controller.upload_task import manager as upload_task_manager
+from submodules.model import enums
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+__engine = None
+
+
+def get_workflow_db_engine():
+    global __engine
+    if __engine is None:
+        __engine = create_engine(os.getenv("WORKFLOW_POSTGRES"))
+    return __engine
```

Comment:
database connection/engine management seems not optimal placed in an api file, i would expect some sort of layer for this or at least an util file?

Reply:
created an adapter file in utils for this